### PR TITLE
feat: GEOT-394: main page

### PR DESCRIPTION
**Pull request by ra8ga**

### CODE REVIEW

File: packages/app-andrew/src/components/group-settings/Basic/styles.scss [hunk 1]
```diff
@@ -1,42 +1,41 @@
 @import '~@gtms/styles/scss/mixins/buttons.scss';
+@import '~@gtms/styles/scss/mixins/fontSize.scss';
+@import '~@gtms/styles/scss/mixins/fontWeight.scss';
 
-.bools {
-  display: flex;
-  flex-direction: column;
+.toremove {
+  display: none;
```
Comment (ra8ga) [inline packages/app-andrew/src/components/group-settings/Basic/styles.scss:6]: functionality is duplicated so I've hidden it

Comment (mariusz-kabala) [reply packages/app-andrew/src/components/group-settings/Basic/styles.scss:6]: ok, I will do a cleanup in a separate PR

File: packages/app-andrew/src/components/group-settings/Tags/PromotedTags/styles.scss [hunk 2]
```diff
@@ -29,7 +30,7 @@
         height: 100%;
         border: 4px solid $theme4;
         background: rgba($theme4, 0.2);
-        border-radius: $borderRadius;
+        bborderradius: $borderRadius;
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/components/group-settings/Tags/PromotedTags/styles.scss:33]: what is bborderradius?

File: packages/app-andrew/src/components/group-settings/Tags/PromotedTags/styles.scss [hunk 3]
```diff
@@ -69,7 +70,7 @@
 
       img,
       picture {
-        border-radius: $borderRadius;
+        bborderradius: $borderRadius;
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/components/group-settings/Tags/PromotedTags/styles.scss:73]: bborderradius - the same here

File: packages/app-andrew/src/components/group-settings/Tags/PromotedTags/styles.scss [hunk 4]
```diff
@@ -83,46 +84,46 @@
...
-
-        .no {
-          &:hover {
-            background: red;
-          }
-        }
-      }
-    }
-  }
+  // .content {
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/components/group-settings/Tags/PromotedTags/styles.scss:87]: remove this

File: packages/app-andrew/src/pages/group-members/index.tsx [hunk 5]
```diff
@@ -21,133 +21,133 @@ export const GroupMembersPage: NextPage<{}> = () => {
               image={'/images/temp_images/avatar-1.png'}
             />
           )}
-          <UserCardMini
+          <GridCard
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/group-members/index.tsx:24]: finally a name that make some sense 👍

File: packages/app-andrew/src/pages/group/[...params].tsx [hunk 6]
```diff
@@ -190,130 +201,158 @@ const GroupPage: NextPage<GroupPageProps> = (props) => {
...
                 </div>
+                <div className={styles.searchInput}>
+                  <div className={styles.search}>
+                    <Button
+                      onClick={() => {
+                        if (!showPromoted) {
+                          setTimeout(() => {
+                            window.scroll({
+                              top:
+                                promotedTagsRef?.current?.offsetTop -
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/group/[...params].tsx:254]: this look wrong, and why do you need a timeout on button onclick callback?

File: packages/app-andrew/src/pages/group/[...params].tsx [hunk 7]
```diff
@@ -190,130 +201,158 @@ const GroupPage: NextPage<GroupPageProps> = (props) => {
...
-                      activeTags={state.activeTags || []}
-                      post={state.activePost}
-                    />
-                  )}
-                </div>
-              </div>
+                    <div>
+                      {!state.activePost && (
+                        <div className={styles.noPostPicture}>
+                          <Picture
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/group/[...params].tsx:336]: I suggest to create a separate component with image, description and whatever is needed

File: packages/app-andrew/src/pages/group/[slug]/styles.scss [hunk 8]
```diff
@@ -45,10 +44,20 @@
     }
   }
 
+  .general {
+    .avatarAndName {
+      // display: flex;
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/group/[slug]/styles.scss:49]: remove this

File: packages/app-andrew/src/pages/index.tsx [hunk 9]
```diff
@@ -1,63 +1,78 @@
...
               {t('btn')}
             </Button>
-            {t('subheadertwo')}
-          </p>
+          </div>
+          <div className={cx(styles.section, styles.recentlyCreatedGroups)}>
+            <RecentlyCreatedGroups groups={groups} />
+          </div>
+          <div className={cx(styles.section, styles.recentlyRegisteredUsers)}>
+            {/* <h2 className={styles.header}>{t('header')}</h2> */}
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/index.tsx:71]: remove this

File: packages/app-andrew/src/pages/styles.scss [hunk 10]
```diff
@@ -1,29 +1,97 @@
...
-    padding: 10px;
-    margin: 10px 0 10px 10px;
-    border-radius: $borderRadius;
-    background: $theme0;
-    display: flex;
-
-    .nav {
-      position: fixed;
+.pageWrapper {
+  // @include pageWrapper();
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/pages/styles.scss:11]: remove it

File: packages/ui/PostSingle/index.tsx [hunk 11]
```diff
@@ -56,7 +56,7 @@ export const PostSingle: FC<{
   onLoginRequest,
   renderFavs,
   favs = [],
-  allowToRespond = true,
+  allowToRespond = false,
```
Comment (mariusz-kabala) [inline packages/ui/PostSingle/index.tsx:59]: and why you changed this?

File: packages/ui/RecentlyAddedPosts/styles.scss [hunk 12]
```diff
@@ -2,7 +2,20 @@
 
 .wrapper {
   .post {
+    padding: 8px;
```
Comment (mariusz-kabala) [inline packages/ui/RecentlyAddedPosts/styles.scss:5]: why not 10?

File: packages/ui/Tag/styles.scss [hunk 13]
```diff
@@ -6,7 +6,7 @@
   padding: 3px 6px;
   border-radius: $borderRadius;
   background: $theme2;
-  color: $theme7;
+  color: $black;
```
Comment (mariusz-kabala) [inline packages/ui/Tag/styles.scss:9]: uuu and a name of the color that actually means something 👍

File: packages/ui/TagsGrid/index.tsx [hunk 14]
```diff
@@ -104,15 +106,11 @@ export const TagsGrid: FC<{
           <ul className={styles.items}>
             {mock.map((item) => (
               <li className={styles.item} key={item.id}>
-                <Picture
-                  additionalStyles={styles.image}
-                  jpg={item.image}
-                  maxHeight={250}
+                <GridCard
+                  name={item.name}
+                  desc={distinctUntilArrayItemChanged.description}
```
Comment (mariusz-kabala) [inline packages/ui/TagsGrid/index.tsx:111]: wtf is that? `distinctUntilArrayItemChanged`

File: packages/app-andrew/src/components/group-settings/Tags/PromotedTags/styles.scss (hunk 2 above)
Comment (ra8ga) [reply packages/app-andrew/src/components/group-settings/Tags/PromotedTags/styles.scss:33]: removed

File: packages/app-andrew/src/components/group-settings/Tags/PromotedTags/styles.scss (hunk 3 above)
Comment (ra8ga) [reply packages/app-andrew/src/components/group-settings/Tags/PromotedTags/styles.scss:73]: removed

File: packages/app-andrew/src/components/group-settings/Tags/PromotedTags/styles.scss (hunk 4 above)
Comment (ra8ga) [reply packages/app-andrew/src/components/group-settings/Tags/PromotedTags/styles.scss:87]: removed

File: packages/app-andrew/src/pages/group-members/index.tsx (hunk 5 above)
Comment (ra8ga) [reply packages/app-andrew/src/pages/group-members/index.tsx:24]: GPT-3 generated that, I named it 'UserMiniBoxWithImageAndText'

File: packages/app-andrew/src/pages/group/[...params].tsx (hunk 7 above)
Comment (ra8ga) [reply packages/app-andrew/src/pages/group/[...params].tsx:335]: http://geotags.atlassian.net/browse/GEOT-398

File: packages/app-andrew/src/pages/group/[slug]/styles.scss (hunk 8 above)
Comment (ra8ga) [reply packages/app-andrew/src/pages/group/[slug]/styles.scss:49]: removed

File: packages/ui/PostSingle/index.tsx (hunk 11 above)
Comment (ra8ga) [reply packages/ui/PostSingle/index.tsx:59]: sorry my bad, I just tested sth, restored proper value

File: packages/ui/RecentlyAddedPosts/styles.scss (hunk 12 above)
Comment (ra8ga) [reply packages/ui/RecentlyAddedPosts/styles.scss:5]: ok I've changed it to 10, but there will be another pull request with some values like 6px, 8px - as there are some exceptions where these two pixels matters for designs

File: packages/ui/Tag/styles.scss (hunk 13 above)
Comment (ra8ga) [reply packages/ui/Tag/styles.scss:9]: GPT-3 generated that

**Pull request merged**